Makefile

```
@@ -295,6 +298,10 @@ test-check-containers:
 tidy: tidy-js
 	tools/tidy
 
+.PHONY: test-containers-compose
+test-containers-compose:
+	for i in $(dir $(wildcard container/**/docker-compose.yaml)); do (cd $$i; sudo docker-compose up -d) || exit 1; done
```

~~I'm wondering about the `|| exit 1`. It seems useless unless you really wanted to enforce the exit code `1` in favor of any other non-zero exit code. Or did you wanted to break the loop here? That's not possible this way because `(…)` forks and the `exit` only affects the fork.~~ Ok, I see that you've just updated the commit so it should be fine now.

---

By the way, **if the `sudo` version in this env is new enough**:
```suggestion
	for i in $(dir $(wildcard container/**/docker-compose.yaml)); do sudo --chdir="$$i" docker-compose up -d; done
```

---

You could check it out if it works by using a container environment using the same image. I suggest to reserve this change for later though. Let's focus on getting the PR in quickly